source/gui/settingsDialogs.py

```diff
@@ -148,10 +148,10 @@ def __init__(self, parent,
 	def _enterTriggersOnOk_ctrlSTriggersOnApply(self, evt):
 		"""Listens for keyboard input and triggers ok button on enter and triggers apply button when control + S is
 		pressed. Cancel behavior is built into wx"""
-		if evt.KeyCode == wx.WXK_RETURN:
-			self.onOk(evt)
+		if evt.KeyCode in (wx.WXK_RETURN, wx.WXK_NUMPAD_ENTER):
+			self.ProcessEvent(wx.CommandEvent(wx.wxEVT_COMMAND_BUTTON_CLICKED, wx.ID_OK))

```

Comment:
I'm ok with this change, however I think the function name / docs need to change.

Reply:
The doc string already looked ok to me. Strictly spoken, it was wrong in the beginning, since it only triggered the OnOk and OnApply handlers and didn't trigger the actual buttons.